screens/UserListScreen.tsx

```diff
@@ -49,6 +66,10 @@ export const UserListScreen = () => {
     );
   }
 
+  const onPressUserDetails = (userdata: any) => {
+    navigation.navigate("UserDetailsScreen", { userdata: userdata });
```

Comment:
もしstateの`userData`と引数の命名が被るのが嫌ならこんな感じでも良いかも
```suggestion
  const onPressUserDetails = (data: any) => {
    navigation.navigate("UserDetailsScreen", { userData: data });
```